src/Workspaces/CoreTest/WorkspaceTests/WorkspaceReferenceTests.cs

```diff
@@ -0,0 +1,203 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.UnitTests.WorkspaceTests
+{
+    public class WorkspaceReferenceTests
+    {
+        [Fact]
+        public async Task PEReferenceTest()
```

Comment:
adding file doesnt matter actually, anything that makes new snapshot to created which doesnt concern reference. it could be updating tree or text or compilation options and etc as long as it doesnt change reference itself.